LibraryWebApp/Models/Book.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LibraryWebApp.Models
+{
+    public class Book
+    {
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        public string Description { get; set; }
+        [Required]
+        public string Title { get; set; }
+        public int Amount { get; set; }
```

Comment:
-optional-

While this code is ok It is a good practice to be explicite about atributes. I would add [Required] to all non optional db fields